codecs/resize/src/lib.rs

```diff
@@ -133,8 +133,9 @@ pub fn resize(
                 unprocessed_output_image[4 * i + 3],
             ));
         }
-        output_image[4 * i + 3] =
-            (unprocessed_output_image[4 * i + 3] * 255.0).clamp(0.0, 255.0) as u8;
+        output_image[4 * i + 3] = (unprocessed_output_image[4 * i + 3] * 255.0)
+            .round()
+            .clamp(0.0, 255.0) as u8;
```

Comment:
Maybe I'd invert the order here to `(... as u8).clamp(0, 255)` so that it doesn't operate on floats anymore, but that's merely a microoptimisation and not important :)

Reply:
Lol no this would be even wrong.